### PR TITLE
feat: core dao

### PR DESCRIPTION
**Pull request by leoslr**
#### Supports for CORE chain.

Contract deployed: 
- [**RequestDeployer**](https://scan.coredao.org/address/0xE99Ab70a5FAE59551544FA326fA048f7B95A24B2#code)
- [**ERC20Proxy**](https://scan.coredao.org/address/0x88Ecc15fDC2985A7926171B938BB2Cd808A5ba40#code)
- [**ERC20FeeProxy**](https://scan.coredao.org/address/0x399F5EE127ce7432E4921a61b8CF52b0af52cbfE#code)
- [**EthereumProxy**](https://scan.coredao.org/address/0x171Ee0881407d4c0C11eA1a2FB7D5b4cdED71e6e#code)
- [**EthereumFeeProxy**](https://scan.coredao.org/address/0xe11BF2fDA23bF0A98365e1A4c04A87C9339e8687#code)

All deployed contracts have been verified.

#### Update contract verification lib.

### CODE REVIEW

**Review by KolevDarko — APPROVED** [review]
👍

**Review by MantisClone — APPROVED** [review]
One minor comment. Looks good :+1:

File: packages/payment-detection/src/eth/multichainExplorerApiProvider.ts
[the remark this answers is not in the document]
Comment (leoslr) [reply packages/payment-detection/src/eth/multichainExplorerApiProvider.ts:1]: @MantisClone I made the update. Not sure the urls are correct for Mantle though. It looks like they're the explorer URLs and not the explorer API URLs.

[the remark this answers is not in the document]
Comment (MantisClone) [reply packages/payment-detection/src/eth/multichainExplorerApiProvider.ts:1]: Fixed: https://github.com/RequestNetwork/requestNetwork/pull/1225

**Pull request merged**